Schema/V1/no.ks.fiks.politisk.behandling.v1.felles.skjerming.schema.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "no.ks.fiks.politisk.behandling.v1.felles.skjerming.schema.json",
+    "title": "Skjerming",
+    "definitions": {},
+    "type": "object",
+    "properties": {
```

Comment:
Lettere å se mangler nå. Feltet tilgangsrestriksjon (M500 i Noark 5, kodefelt her) må legges til som required.
Tilgangsrestriksjon angir regler for skjerming / at noe ikke er offentlig tilgjengelig.
Skjermingshjemmel er referanse til regelverket som er grunnlaget for påført tilgangsrestriksjon.

Reply:
Ok. Lagt til feltet "tilgangsrestriksjon" slik i skjerming skjema som required felt:
```
"tilgangsrestriksjon": {
            "type": "object",
            "properties": {
                "kodeverdi": {
                    "type": "string"
                },
                "kodebeskrivelse": {
                    "type": "string"
                }
            }
        }
```